lib/std/log.zig

```diff
@@ -100,6 +100,23 @@ pub const Level = enum {
     info,
     /// Debug: messages only useful for debugging.
     debug,
+
+    /// Returns a string literal of the given level in full text form.
+    pub fn asText(comptime self: Level) switch (self) {
```

Comment:
Why not implement a custom `format`?

Reply:
This is used to append to other comptime strings, i.e.
```zig
level.asText() ++ ": " ++ fmt ++ "\n"
```

Reply:
Ah, I see. The repetition in the return type and the function body is not that nice but I can't think of a way to DRY it up.